components/network/samba/manifests/sample-manifest.p5m

```diff
@@ -1,27 +1,3 @@
-#
```

Comment:
Don't touch this file, it didn't change.

Reply:
Thanks. I wondered why it should have changed, but this time I at least did not forget to create that manifest... ;o)